components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/AuthContextAPIClient.java

```diff
@@ -76,7 +76,7 @@ public static String getContextProperties(String backendURL) {
             }
             response = builder.toString();
         } catch (IOException e) {
-            log.error("Sending " + HTTP_METHOD_GET + " request to URL : " + url + "failed.", e);
+            log.error("Sending " + HTTP_METHOD_GET + " request to URL : " + url + " " +"failed.", e);
```

Comment:
```suggestion
            log.error("Sending " + HTTP_METHOD_GET + " request to URL : " + url + ", failed.", e);
```
On top of your fix, shall we improve in this manner? The code would be more cleaner and when printed on the console, the comma will show the place where the parameter ends.

Reply:
Fixed. 990db06